cg/cli/demultiplex/copy_novaseqx_data.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+import os
+import shutil
+from typing import List, Optional
+
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def is_copied(analysis_directory: Path) -> bool:
+    return Path(analysis_directory, DemultiplexingDirsAndFiles.COPY_COMPLETE).exists()
+
+
+def is_analyzed(analysis_directory: Path) -> bool:
+    return Path(
+        analysis_directory,
+        DemultiplexingDirsAndFiles.DATA,
+        DemultiplexingDirsAndFiles.ANALYSIS_COMPLETED,
+    ).exists()
+
+
+def is_in_demultiplexed_runs(flow_cell_name: str, demultiplexed_runs: Path) -> bool:
```

Comment:
I would include a docstring here explaining what is `flow_cell_name`. That could be confusing if it is the full name or just the id

Reply:
I agree it is confusing, changed the signature to use the flow cell path instead